src/apps/concepts/__test__/redux/reducer.test.ts

```diff
@@ -0,0 +1,65 @@
+import {reducer} from "../../redux/reducer";
+import {APIConcept, ConceptsState} from "../../types";
+import {RETRIEVE_ACTIVE_CONCEPTS_ACTION} from "../../redux/actionTypes";
+
+const testConcept: APIConcept = {
+    display_name: "testConcept",
+    url: "url",
+    version_url: "versionUrl",
+    mappings: [],
+    retired: false,
+    source_url: "sourceUrl",
+    id: "testId",
+    external_id: "extId",
+    concept_class: "",
+    datatype: "TEXT",
+    names: [],
+    descriptions: [],
+    extras: {}
+};
+
+const initialState: ConceptsState = {
+    mappings: [],
+    concepts: {
+        items: [],
+        responseMeta: undefined
+    },
+    activeConcepts: {
+        items: [],
+        responseMeta: undefined
+    }
+};
+
+describe("RETRIEVE_ACTIVE_CONCEPTS_ACTION", () => {
+    it('should return initial state when empty payload is passed', () => {
+        const startAction = {
+            type: RETRIEVE_ACTIVE_CONCEPTS_ACTION,
+            action: {
+                payload: [],
+                responseMeta: undefined
+            }
+        };
+        expect(reducer(initialState, startAction)).toEqual(initialState);
+
+    });
+
+    it('should return updated state with given payload', () => {
+        const startAction = {
+                type: RETRIEVE_ACTIVE_CONCEPTS_ACTION,
+                payload: [testConcept],
+                responseMeta: { num_found: 1 }
+            }
+        ;
+        const expectedState = {
+            mappings: [],
+            concepts: {
+                items: []
+            },
+            activeConcepts: {
+                items: [testConcept],
+                responseMeta: { num_found: 1 }
+            }
+        };
+        expect(reducer(initialState, startAction)).toEqual(expectedState);
+    });
+});
```

Comment:
can we fix the console error coming while running this test 
```  ● Console

    console.error node_modules/redux/lib/redux.js:325
      No reducer provided for key "concepts"

Reply:
This is not impacting the test cases. This issue is coming because of existing reducers and not because of the reducers which are newly added. It requires refactoring of existing reducers.